.github/workflows/board-move-issues.yml

```diff
@@ -10,18 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: add-new-issues-to-repository-based-project-column
-      uses: takanabe/github-actions-automate-projects@v0.0.2
-      if: github.event_name == 'issues' && github.event.action == 'labeled' && github.event.label.name == 'analytics research'
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        GITHUB_PROJECT_URL: https://github.com/cal-itp/data-infra/projects/6
-        GITHUB_PROJECT_COLUMN_NAME: Backlog
-
     - name: add-new-issues-to-repository-based-project-column
       uses: takanabe/github-actions-automate-projects@v0.0.2
       if: github.event_name == 'issues' && github.event.action == 'labeled' && github.event.label.name == 'new team member'
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        GITHUB_PROJECT_URL: https://github.com/cal-itp/data-infra/projects/6
+        GITHUB_PROJECT_URL: https://github.com/cal-itp/data-infra/projects#workspaces/data-services-current-sprint-620d721f9d810a00186ed9c9/board
```

Comment:
This link appears to be for Zenhub, so I don't think this will work properly.

Reply:
yeah that's a great point @evansiroky  - I was going to try to keep the `new team member` label workflow and see if the zenhub link worked with it but it seems unlikely. I just removed the workflow and re-requested your review if you don't mind giving it an approval when you get a chance